examples/configurables/configurable.description

```diff
@@ -0,0 +1,10 @@
+// The configurable variables feature allows configuration of module level
+// basic variables at the program execution. This enables user to initialize
+// variables with externally provided values. These variables can be
+// initialized using the `configurable` keyword.
```

Comment:
Include `Configured values would always be a subtype of readonly&anydata.` Also, add `The configurable keyword can be applied to a module-level variable declaration that has an initializer`